toolset/benchmark/benchmarker.py

```diff
@@ -561,6 +561,7 @@ def exit_with_code(code):
             sudo service postgresql restart
             sudo service cassandra restart
             /opt/elasticsearch/elasticsearch restart
+   	    sudo /etc/init.d/apache2 stop
```

Comment:
Awesome. Could you push a fix removing the line? Other than that I'd say this is good to merge!